src/app/forms/calendar-page/calendar-page.component.ts

```diff
@@ -25,6 +25,12 @@ export class CalendarPageComponent {
   dateChanged(val) {
     console.log('date changed!', val);
   }
+  startDateChanged(val) {
+    console.log('start date changed!', val);
+  }
+  endDateChanged(val) {
+    console.log('end date changed!', val);
+  }
 
```

Comment:
Can we please combine these callbacks into one for range and have the callback report both the start and end date changed? `(onRangeSelect)` possibly returning `{start: Date, end: Date}`. 

This cuts down on the necessary boilerplate and allows the end user to react to changes to both in the same callback by default.

Reply:
Changed Accordingly